events/lifestyles/warfare_lifestyle/martial_strategy_events.txt

```diff
@@ -363,6 +363,38 @@ martial_strategy.1051 = {
 		add_prestige = medium_prestige_gain
 		add_dread = minor_dread_gain
 	}
+	option = {
+		name = martial_strategy.1051.g.wc
+		trigger = {
+			AND = {
+				has_trait = sadistic
+				has_trait = arbitrary
+			}
+		}
+		trait = sadistic
+		trait = arbitrary
+
+		set_nickname_effect = { NICKNAME = nick_the_ice_hearted }
+		add_dread = major_dread_gain
+		add_prestige = minor_prestige_gain
```

Comment:
I think callous would make more sense. Ice-hearted is basically a synonym of callous.
```suggestion
		trigger = {
			has_trait = callous
		}
		trait = callous

		set_nickname_effect = { NICKNAME = nick_the_ice_hearted }
		add_dread = minor_dread_gain
		add_prestige = medium_prestige_gain
```

Reply:
done in 7ce343d0